components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -799,9 +799,25 @@ function ImageBuildView(props: ImageBuildViewProps) {
                 <WorkspaceLogs logsEmitter={logsEmitter} errorMessage={props.error?.message} />
             </Suspense>
             {!!props.onStartWithDefaultImage && (
-                <button className="mt-6 secondary" onClick={props.onStartWithDefaultImage}>
-                    Continue with Default Image
-                </button>
+                <>
+                    <div className="mt-6 w-11/12 lg:w-3/5">
+                        <p className="text-center text-gray-400 dark:text-gray-500">
+                            <span className="font-bold">Did you know?</span> You can use <code>gp rebuild</code> to
+                            validate and debug your custom workspace image before pushing &nbsp;&middot;&nbsp;
+                            <a
+                                href="https://www.gitpod.io/docs/configure/workspaces/workspace-image#trying-out-changes-to-your-dockerfile"
+                                target="_blank"
+                                rel="noopener noreferrer"
+                                className="gp-link"
+                            >
+                                Read More
```

Comment:
**Suggestion:** To make it consistent with [existing `Learn More`](https://github.com/gitpod-io/gitpod/blob/e147452a00b53b3ce35d58f0292ea85951cc5ee7/components/dashboard/src/settings/Preferences.tsx#L56-L63) callouts.

```suggestion
                                Learn More
```

Reply:
**praise**: Thanks for caring about _help links_ consistency, @Siddhant-K-code!